RELEASES.md

```diff
@@ -83,14 +83,14 @@ We reserve the right to drop support for releases if they are deemed unused (for
 
 |Release|End of Life Date|
 |-------|----------------|
-|`v0.1.x+ibc-go-v7.3.x`|September 17, 2024|
+|`v0.1.x+ibc-go-v7.3.x`|March 17, 2025|
```

Comment:
Is this one not retracted? 🤔

Reply:
Indeed, this release has not been explicitly retracted. Probably it should be, since ibc-go v7.3.x is retracted.